elephant/asset.py

```diff
@@ -267,70 +267,58 @@ def intersection_matrix(
     either identical or completely disjoint). Then calculate the
-    intersection matrix M of the two binned data, where M[i,j] is the overlap
-    of bin i in the first binned data and bin j in the second binned data
-    (i.e. the number of spike trains spiking both at bin i and at bin j).
-    The matrix entries can be normalized to values between 0 and 1 via
+    intersection matrix `M` of the two binned data, where `M[i,j]` is the overlap
+    of bin `i` in the first binned data and bin `j` in the second binned data
+    (i.e. the number of spike trains spiking both at bin `i` and at bin `j`).
+    The matrix entries can be normalized to values between `0` and `1` via
     different normalizations (see below).
 
     Parameters
     ----------
     spiketrains : list of neo.SpikeTrain
-        list of SpikeTrains from which to compute the intersection matrix
+        A list of SpikeTrains from which to compute the intersection matrix.
     binsize : pq.Quantity
-        size of the time bins used to define synchronous spikes in the given
+        The size of the time bins used to define synchronous spikes in the given
         SpikeTrains.
     spiketrains_y : list of neo.SpikeTrain, optional
-        list of SpikeTrains for the second time dimension
-        if unspecified, spiketrains is used for both
+        A list of SpikeTrains for the second time dimension.
+        If unspecified, `spiketrains` is used for both.
         Default: None
-    t_start_x : pq.Quantity, optional
-        start time of the binning for the first axis of the intersection
+    t_start_x, t_start_y : pq.Quantity, optional
+        The start time of the binning for the first and second axes of the intersection
         matrix, respectively.
-        If None (default) the attribute t_start of the SpikeTrains is used
+        If None (default) the attribute `t_start` of the SpikeTrains is used
         (if the same for all spike trains).
         Default: None
-    t_start_y : pq.Quantity, optional
-        start time of the binning for the second axis of the intersection
-        matrix
-        If None (default) the attribute t_start of the SpikeTrains is used
-        (if the same for all spike trains).
-        Default: None
-    t_stop_x : pq.Quantity, optional
-        stop time of the binning for the first axis of the intersection
+    t_stop_x, t_stop_y : pq.Quantity, optional
+        The stop time of the binning for the first and second axes of the intersection
         matrix, respectively.
         If None (default) the attribute t_stop of the SpikeTrains is used
         (if the same for all spike trains).
         Default: None
-    t_stop_y : pq.Quantity, optional
-        stop time of the binning for the second axis of the intersection
-        matrix
-        If None (default) the attribute t_stop of the SpikeTrains is used
-        (if the same for all spike trains).
-        Default: None
     norm : int, optional
```

Comment:
This could be changed to `normalization`.
Also, 0 or None have the same effect.
One option would be to use strings and give the meaning of each normalization operation.

Reply:
@stellalessandra @Kleinjohann what do you think?

Reply:
No strong opinion. If we can come up with short descriptive strings that would be good. `None` should still work and be the default though.